_config.yaml

```diff
@@ -35,7 +35,11 @@ header_pages:
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
   - workshop/workshop-materials.md
+  - workshop/resources-for-consultation-sessions.md
```

Comment:
Can we change the `nav_title` on this page too? Right now it is "Consultation Sessions", but I think "Resources" might be a better title. Others may disagree!

Reply:
Sounds good. Also done!